### ISSUE TITLE
Hard configuration error when no javac

### ISSUE THREAD

**Issue by ampli** [opencog/link-grammar#812]
I'm now it the process of testing LG on FreeBSD.
On my initial configuration Java is not installed.
When I configure (default configuration - w/o configuration options) `configure` fails with this error:
```
configure:19480: checking for javac
configure:19511: result: no
configure:19520: error: cannot find JDK; try setting $JAVAC or $JAVA_HOME
```

This is a hard error generated in AX_JNI_INCLUDE_DIR.
Should I convert it to a warning only? (Like what is done for AX_PYTHON_DEVEL.)

**Comment by linas** [comment]
Yes, it was always meant to be a soft error.  The Java bindings are pretty useless to most people.

**Comment by ampli** [comment]
Fix in PR #813.
